app/components/channel_list/header/index.tsx

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Text, View} from 'react-native';
+
+import CompassIcon, {COMPASS_ICONS} from '@app/components/compass_icon';
+import TouchableWithFeedback from '@app/components/touchable_with_feedback';
+import {useTheme} from '@app/context/theme';
+import {typography} from '@app/utils/typography';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
```

Comment:
```suggestion
import CompassIcon, {COMPASS_ICONS} from '@components/compass_icon';
import TouchableWithFeedback from '@components/touchable_with_feedback';
import {useTheme} from '@context/theme';
import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
import {typography} from '@utils/typography';
```

Same comment on the COMPASS_ICONS enum